react/getting_started_with_react/keys_in_react.md

```diff
@@ -57,22 +57,22 @@ function TodoList() {
 }
 ~~~
 
-<span id="index-as-key">Additionally, if the list is static and will remain unchanged throughout the application's life, we can use the array index as a key. Though you should be careful when working with `index` as key.</span>
+<span id="index-as-key">Additionally, if you're sure the list will remain unchanged throughout the application's life,  you can use the array index as a key. But this is not recommended, since it can lead to confusing bugs if the list *does* change with any items deleted, inserted or rearranged. You will learn more about this in the assignment section's linked articles.</span>
```

Comment:
```suggestion
<span id="index-as-key">Additionally, if you're sure the list will remain unchanged throughout the application's life,  you can use the array index as a key. However, this is not recommended since it can lead to confusing bugs if the list changes when items are deleted, inserted, or rearranged. You will learn more about this in the assignment section's linked article.</span>
```

Reply:
I should have paid better attention when I gave the previous feedback. I'm sorry about coming with yet another suggestion. Thanks for your time!